src/domain/shared/utils/spaceLevel.ts

```
@@ -0,0 +1,14 @@
+import { SpaceLevel } from '../../../core/apollo/generated/graphql-schema';
+import { JourneyTypeName } from '../../journey/JourneyTypeName';
+
+export const getChildJourneyTypeName = ({ level }: { level: SpaceLevel }): JourneyTypeName => {
+  switch (level) {
+    case SpaceLevel.Challenge:
+      return 'subspace' as JourneyTypeName;
+    case SpaceLevel.Opportunity:
+      return 'subsubspace' as JourneyTypeName;
+    case SpaceLevel.Space:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)